substratevm/src/com.oracle.graal.pointsto.standalone/src/com/oracle/graal/pointsto/standalone/PointsToAnalyzer.java

```diff
@@ -197,6 +201,14 @@ private PointsToAnalyzer(String mainEntryClass, OptionValues options) {
             NoClassInitializationPlugin classInitializationPlugin = new NoClassInitializationPlugin();
             plugins.setClassInitializationPlugin(classInitializationPlugin);
             aProviders.setGraphBuilderPlugins(plugins);
+            registerInvocationPlugins(originalProviders.getSnippetReflection(), plugins.getInvocationPlugins(), originalProviders.getReplacements(),
+                            false, true, false, originalProviders.getLowerer());
+            /** Disable inlineBeforeAnalysis for two reasons:
+             *  1) It is necessary for standalone analysis;
+             *  2) Standalone analysis uses {@link com.oracle.graal.pointsto.phases.InlineBeforeAnalysisPolicy#NO_INLINING}
+             *  as inline policy, but it's not well supported since https://github.com/oracle/graal/commit/cc2ad502c6877a2c11685491c6609f0ff8ce7b14.
+             **/
```

Comment:
How does that commit break `NO_INLINING` policy? cc: @christianwimmer

Reply:
For the changes below:
![image](https://user-images.githubusercontent.com/6834162/218364029-62c51c96-063e-420d-9c3c-1bdb4c09211e.png)
Given a `caller` with the following states: 
![image](https://user-images.githubusercontent.com/6834162/218364448-780119bf-7542-4e82-a8a1-fe817131d965.png)

As the `caller` is not null but `caller.caller` is null. Before change, it goes to the line 77 on the left side. After change, it goes to the line 76 on the right side. When the `policy` here is `InlineBeforeAnalysisPolicy#NO_INLINING`, it finally goes to `InlineBeforeAnalysis#openCalleeScope` which throws an unimplemented error.

Reply:
@ziyilin I'm not sure that I understand what the problem is. Can you provide a patch that fixes `InlineBeforeAnalysisPolicy#NO_INLINING`?

Reply:
This problem is fixed in the latest master. I have deleted the option setting code.